addons/l10n_in_ewaybill_stock/models/stock_move.py

```diff
@@ -0,0 +1,64 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+    _description = "Stock Move Ewaybill"
+
+    ewaybill_id = fields.Many2one(
+        comodel_name='l10n.in.ewaybill'
+    )
+
+    ewaybill_company_currency_id = fields.Many2one(
+        comodel_name='res.currency',
+        related='company_id.currency_id'
+    )
+
+    # Need to store values because we send it to the ewaybill and we need to keep the same value
+    ewaybill_price_unit = fields.Monetary(
+        compute='_compute_ewaybill_price_unit',
+        currency_field='ewaybill_company_currency_id',
+        store=True
+    )
+    ewaybill_tax_ids = fields.Many2many(
+        comodel_name='account.tax',
+        string="Taxes",
+        compute='_compute_tax_ids',
+        store=True
+    )
+
+    @api.depends('product_id.uom_id', 'product_id.standard_price', 'ewaybill_id')
+    def _compute_ewaybill_price_unit(self):
+        for line in self:
+            if line.ewaybill_id:
+                if line.ewaybill_id.state == 'pending':
+                    line.ewaybill_price_unit = line.product_id.uom_id._compute_price(
+                        line.product_id.with_company(line.company_id).standard_price, line.product_uom
```

Comment:
Standard Price because we need to get it by the price we sell the product not on the valuation

Reply:
My bad it should be the standard price (cost price) since here it is used for the transfers.
Suffering from cold and cough seems like got into my head :cold_face: :mask: